portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,31 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    String[] quotes = {"You know more than you think you know, just as you know less than you want to know.", 
+                        "One day I will find the right words, and they all will be simple.",
+                        "I assure you that the world is not so amusing as something we imagined."};
+
+    String json = convertToJson(quotes);
```

Comment:
This isn't needed for the assignment but maybe something fun to think about: passing all the quotes data will likely not scale (say if we had millions of quotes). Perhaps it'd be better to move the getRandomQuote method here in the server code, or have some pagination parameter.

Reply:
Thank you for the feedback! I will make sure to work on this by week 4.